backend/src/db/migrations/20230823072025_add_is_survey_completed_column_to_users_table.ts

```diff
@@ -0,0 +1,21 @@
+import { type Knex } from 'knex';
+
+const TABLE_NAME = 'users';
```

Comment:
We decided to change users to user_details table)

Reply:
@OlenaYatseniuk  No, we mean is_survey_completed will be in user_details table, not in users

Reply:
@fshabanov So, should I wait for the merge of pr #46 ? Because I need this table to change my functionality from user to user_details ?

Reply:
Yes, let's fix other issues in this ticket for now, we hope to merge #46 tomorrow in the morning